cmd/create/operatorroles/cmd.go

```diff
@@ -180,11 +180,6 @@ func run(cmd *cobra.Command, argv []string) error {
 		cluster = r.FetchCluster()
 	}
 
-	if args.prefix == "" && args.sharedVpcRoleArn != "" {
```

Comment:
should we continue to support those parameters, but only prompt that those parameters already deprecreated?

Reply:
We still use both parameters for creating operator roles prior to the cluster creation.
The main flow for shared VPC is creating the operator roles using these two parameters.
In addition, it would be possible to create the operator roles after cluster creation.